common/router_paths.ts

```diff
@@ -9,4 +9,5 @@ export const routerPaths = {
   modelList: '/model-registry/model-list',
   overview: '/overview',
   modelGroup: '/model-registry/model/:id',
+  modelVersion: '/model-registry/model-version/:id',
```

Comment:
Shall we design the URL like `/model-registry/model/:id/v/:version`? We may need the model id parameter in the context, for example, to show the model name in breadcrumb.

Reply:
I am not sure if the model version detail method will return the model name, if yes we won't need the model id.

Reply:
yeah, we can leave it as it is and get back to this once BE API is finalized.